feat(documentation): Add collaborators rights and dashboard managment (#2475)
Scalingo/documentation: src/_posts/platform/app/2000-01-01-collaborators.md

---
title: Collaborators
modified_at: 2024-02-07 00:00:00
tags: app collaborators
---

## Context

> Collaborators allow you to add other accounts to help you manage your application. Collaborators basically have all the features of an owner account. It is also useful if you want to [transfer ownership of an application](https://doc.scalingo.com/platform/app/lifecycle-management#transfer-ownership-of-an-application).

Reviewer: I suggest to re-phrase this like so:

```suggestion
Collaborators is a feature allowing the owner of an app to invite other Scalingo accounts to team up and work jointly on the app. An app can have as many collaborators as needed.

An owner of an app can also [transfer the ownership of the app]({% post_url platform/app/2000-01-01-lifecycle-management %}#transfer-ownership-of-an-app) to a collaborator.

{% note %}
Delegating rights to collaborators should be taken seriously. Even if collaborators can't delete the application, they have enough rights to disrupt the activities or businesses depending on it.
{% endnote %}
```

## Collaborators rights

### Collaborator can do

- Deploy
- Manage containers and addons
- View logs
- Access the review applications
- View metrics
- View activity
- Access the DB Dashboard
- Almost all user actions except those under

### Collaborator can't do

- Delete application
- Rename application
- Transfer application

> ## How to manage collaborators

Author: I have updated my text following your recomandation

### From the dashboard

You will the how to manage collaborators from the CLI there: [manage collaborators of the application](https://doc.scalingo.com/platform/cli/features#manage-collaborators-of-the-application)

- Go to your application that needs a new collaborator
- Go to the Settings tab
- You will automatically be taken to the Collaborators menu
- Click on the Add button
- Add the required collaborator email
- You will now see the collaborator in light grey in the table.
- The collaborator will receive an email to join the collaboration (and subscribe to Scalingo if required).
- Once done, you will see the collaborator in black.

{% note %}
To delete a collaborator, in the collaborator table, simply click on the trash can corresponding to the collaborator's mail and validate in the pop-up.
{% endnote %}

### From the CLI

There you will learn how to manage collaborators from the CLI: [manage collaborators of the application](https://doc.scalingo.com/platform/cli/features#manage-collaborators-of-the-application)

> {% note %}

Reviewer: I suggest to remove this note from here.
I put it back in the very first lines 😃

Author: Like top of the file or before From the dashboard ?

tags: app collaborators
---

Collaborators is a feature allowing the owner of an app to invite other Scalingo accounts to team up and work jointly on the app. An app can have as many collaborators as needed.

An owner of an app can also [transfer the ownership of the app]({% post_url platform/app/2000-01-01-lifecycle-management %}#transfer-ownership-of-an-app) to a collaborator.

{% note %}
Delegating rights to collaborators should be taken seriously. Even if collaborators can't delete the application, they have enough rights to disrupt the activities or businesses depending on it.
{% endnote %}

## Understanding Collaborators Rights

Collaborators can:

- Trigger deployments
- Manage containers (scale up and down, both horizontally and vertically)
- Manage addons (create, delete, change plan)
- View logs
- Access [Review Apps]({% post_url platform/app/2000-01-01-review-apps %})
- View metrics
- View activity
- Access the DB Dashboard(s)
- Manage other collaborators (invite, remove)

Collaborators can't:

- Delete the application
- Rename the application
- Transfer the ownership of the application to another user

## Managing Collaborators

### Listing Collaborators

> Using the Dashboard

Reviewer: ```suggestion
#### Using the Dashboard
```

- Go to your application that needs a new collaborator
- Go to the Settings tab
- You will automatically be taken to the Collaborators menu

> Using the Command Line

Reviewer: ```suggestion
#### Using the Command Line
```

```shell
scalingo --app my-app collaborators
```

### Adding a Collaborator

> Using the Dashboard

Reviewer: ```suggestion
#### Using the Dashboard
```

- Go to your application that needs a new collaborator
- Go to the Settings tab
- You will automatically be taken to the Collaborators menu
- Click on the Add button
- Add the required collaborator email
- You will now see the collaborator in light grey in the table.
- The collaborator will receive an email to join the collaboration (and subscribe to Scalingo if required).
- Once done, you will see the collaborator in black.

> Using the Command Line

Reviewer: ```suggestion
#### Using the Command Line
```

```shell
scalingo --app my-app collaborators-add user@example.com
```

### Removing a Collaborator

> Using the Dashboard

Reviewer: ```suggestion
#### Using the Dashboard
```

- Go to your application that needs a new collaborator
- Go to the Settings tab
- You will automatically be taken to the Collaborators menu
- Click on the trash can corresponding to the collaborator's mail
- Validate in the pop-up

> Using the Command Line

Reviewer: ```suggestion
#### Using the Command Line
```
